flask/app/participants.py

```diff
@@ -37,22 +37,36 @@
 def list_participants():
 
     # parsing query parameters
-    max_rows = request.args.get("max_rows", default=100)
+    limit = request.args.get("limit", default=10)
+    page = request.args.get("page", default=1)
     starts_with = request.args.get("starts_with", default="", type=str)
     starts_with = f"{starts_with}%"  # sql syntax
     order_by_col = request.args.get("order", default="participant_id", type=str)
-    # need some default or we need an ifelse statement, one with order_by method and one without. AFAIK there is no 'default' parameter we can pass into order_by to get the default sql ordering scheme
+    filter_by = request.args.get("filter", default="participant_id,", type=str)
+    filter_by_col, filter_val = filter_by.split(",")[0], filter_by.split(",")[1]
+    filter_val = f"%{filter_val}%"
 
     try:
-        int(max_rows)
+        int(limit)
     except:
-        return "Max rows must be a valid integer", 400
+        return "Limit must be a valid integer", 400
+
+    try:
+        int(page)
+    except:
+        return "Page must be a valid integer", 400
```

Comment:
If the `type` parameter works as it does above with `order_by_col` and `filter_by` then I think we can do `type=int` for the numeric parameters and wrap the `.get` calls in a try-except that catches the [ValueError](https://werkzeug.palletsprojects.com/en/1.0.x/datastructures/#werkzeug.datastructures.MultiDict)

Reply:
This was something I noticed when we were implementing a limit last time-  when type=int is specified and a string is passed instead of int, no error is raised.

Reply:
I see, for some reason it returns None instead of raising a ValueError...